### ISSUE TITLE
Names for external plugins

### ISSUE THREAD

**Issue by frase-git** [cmsimple-xh/cmsimple-xh#559]
The input fields for external plugins are labeled only with "Name" in the settings. In the tooltips it is only then stated that these are external plugin names.
I suggest to rename these input fields in the corresponding meta language files.

metaen.php:
`$mtx['external']="External plugin name";`
metade.php:
`$mtx['external']="Name des externen Plugins";`

Furthermore, the name "Editmenu" for the admin menu can lead to some confusion - which just happened to me ;-) 
Of course, I know that this designation is historical and not so easy to change (too many places).
Probably someone has already noticed this in German, which is why it already says: "_Bearbeitungsmenü (Admin-Menü)_".
I suggest that this should also be more clearly marked in English:
metaen.php:
`$mtx['editmenu']="Editmenu (Admin menu)";`

**Comment by olape-git** [comment]
The problem with the metaXX.php, they are NOT automatically created by the system from a metadefault.php, analog to the language files. So who works under pl, da, sk, ... has nothing of it. There is then simply no metapl.php, etc.

**Comment by frase-git** [comment]
Yes of course, for other languages a corresponding metaXX.php must be created manually. It just needs to be documented properly. (I didn't find anything about this, by the way.)
But this should not stop us to correct at least the two delivered languages.
